### ISSUE TITLE
Tags in the left menu not clickable except for the first one

### ISSUE THREAD

**Issue by viktomas** [laurent22/joplin#3876]
## Environment

Joplin version: Joplin 1.2.4 (prod, darwin)
Platform: macOS
OS specifics: Catalina 10.15.7

## Steps to reproduce

1. have a notebook open
1. have only two tags
2. try to click the second tag **(doesn't work, but should)**
3. click the first tag (that works)
4. now all tags work

![Kapture 2020-10-07 at 12 17 59](https://user-images.githubusercontent.com/1079718/95319510-46c07000-0898-11eb-8a88-16875693bc28.gif)

## Describe what you expected to happen

I would expect the tag to open. But it doesn't.

## Logfile
I followed https://joplinapp.org/debugging/, but there has been no output.

<details>
<summary>Irrelevant logs</summary>

```
/Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:11494 Warning: componentWillReceiveProps has been renamed, and is not recommended for use. See https://fb.me/react-async-component-lifecycle-hooks for details.

* Move data fetching code or side effects to componentDidUpdate.
* If you're updating state whenever props change, refactor your code to use memoization techniques or move it to static getDerivedStateFromProps. Learn more at: https://fb.me/react-derived-state
* Rename componentWillReceiveProps to UNSAFE_componentWillReceiveProps to suppress this warning in non-strict mode. In React 17.x, only the UNSAFE_ name will work. To rename all deprecated lifecycles to their new names, you can run `npx react-codemod rename-unsafe-lifecycles` in your project source folder.

Please update the following components: Connect(TagItemComponent), Connect(TagListComponent)
printWarning @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:11494
lowPriorityWarning @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:11513
ReactStrictModeWarnings.flushPendingUnsafeLifecycleWarnings @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:11680
flushRenderPhaseStrictModeWarningsInDEV @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:23134
commitRootImpl @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:22428
unstable_runWithPriority @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/scheduler/cjs/scheduler.development.js:643
runWithPriority$2 @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:11305
commitRoot @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:22414
runRootCallback @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:21554
(anonymous) @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:11353
unstable_runWithPriority @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/scheduler/cjs/scheduler.development.js:643
runWithPriority$2 @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:11305
flushSyncCallbackQueueImpl @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:11349
flushSyncCallbackQueue @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:11338
scheduleUpdateOnFiber @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:21431
enqueueSetState @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:13100
Component.setState @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react/cjs/react.development.js:325
onStateChange @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-redux/lib/components/connectAdvanced.js:222
notify @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-redux/lib/utils/Subscription.js:30
notifyNestedSubs @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-redux/lib/utils/Subscription.js:69
onStateChange @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-redux/lib/components/connectAdvanced.js:219
notify @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-redux/lib/utils/Subscription.js:30
notifyNestedSubs @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-redux/lib/utils/Subscription.js:69
onStateChange @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-redux/lib/components/connectAdvanced.js:219
notify @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-redux/lib/utils/Subscription.js:30
notifyNestedSubs @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-redux/lib/utils/Subscription.js:69
onStateChange @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-redux/lib/components/connectAdvanced.js:219
dispatch @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/redux/lib/createStore.js:186
generalMiddleware @ /Applications/Joplin.app/Contents/Resources/app.asar/lib/BaseApplication.js:444
generalMiddleware @ /Applications/Joplin.app/Contents/Resources/app.asar/app.js:358
(anonymous) @ /Applications/Joplin.app/Contents/Resources/app.asar/lib/BaseApplication.js:381
dispatch @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/redux/lib/applyMiddleware.js:45
reduxSharedMiddleware @ /Applications/Joplin.app/Contents/Resources/app.asar/lib/components/shared/reduxSharedMiddleware.js:74
async function (async)
reduxSharedMiddleware @ /Applications/Joplin.app/Contents/Resources/app.asar/lib/components/shared/reduxSharedMiddleware.js:71
generalMiddleware @ /Applications/Joplin.app/Contents/Resources/app.asar/lib/BaseApplication.js:452
generalMiddleware @ /Applications/Joplin.app/Contents/Resources/app.asar/app.js:358
(anonymous) @ /Applications/Joplin.app/Contents/Resources/app.asar/lib/BaseApplication.js:381
refreshNotes @ /Applications/Joplin.app/Contents/Resources/app.asar/lib/BaseApplication.js:342
async function (async)
refreshNotes @ /Applications/Joplin.app/Contents/Resources/app.asar/lib/BaseApplication.js:292
generalMiddleware @ /Applications/Joplin.app/Contents/Resources/app.asar/lib/BaseApplication.js:525
async function (async)
generalMiddleware @ /Applications/Joplin.app/Contents/Resources/app.asar/lib/BaseApplication.js:452
generalMiddleware @ /Applications/Joplin.app/Contents/Resources/app.asar/app.js:358
(anonymous) @ /Applications/Joplin.app/Contents/Resources/app.asar/lib/BaseApplication.js:381
tagItem_click @ /Applications/Joplin.app/Contents/Resources/app.asar/gui/SideBar/SideBar.js:251
onClick @ /Applications/Joplin.app/Contents/Resources/app.asar/gui/SideBar/SideBar.js:301
callCallback @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:347
invokeGuardedCallbackDev @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:397
invokeGuardedCallback @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:454
invokeGuardedCallbackAndCatchFirstError @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:468
executeDispatch @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:600
executeDispatchesInOrder @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:622
executeDispatchesAndRelease @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:725
executeDispatchesAndReleaseTopLevel @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:733
forEachAccumulated @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:707
runEventsInBatch @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:750
runExtractedPluginEventsInBatch @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:881
handleTopLevel @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:5901
batchedEventUpdates @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:2344
dispatchEventForPluginEventSystem @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:5996
dispatchEvent @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:6025
unstable_runWithPriority @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/scheduler/cjs/scheduler.development.js:643
runWithPriority$2 @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:11305
discreteUpdates$1 @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:21672
discreteUpdates @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:2359
dispatchDiscreteEvent @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:5979
Show 3 more frames
/Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:11494 Warning: componentWillUpdate has been renamed, and is not recommended for use. See https://fb.me/react-async-component-lifecycle-hooks for details.

* Move data fetching code or side effects to componentDidUpdate.
* Rename componentWillUpdate to UNSAFE_componentWillUpdate to suppress this warning in non-strict mode. In React 17.x, only the UNSAFE_ name will work. To rename all deprecated lifecycles to their new names, you can run `npx react-codemod rename-unsafe-lifecycles` in your project source folder.

Please update the following components: Connect(TagItemComponent), Connect(TagListComponent)
printWarning @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:11494
lowPriorityWarning @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:11513
ReactStrictModeWarnings.flushPendingUnsafeLifecycleWarnings @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:11686
flushRenderPhaseStrictModeWarningsInDEV @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:23134
commitRootImpl @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:22428
unstable_runWithPriority @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/scheduler/cjs/scheduler.development.js:643
runWithPriority$2 @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:11305
commitRoot @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:22414
runRootCallback @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:21554
(anonymous) @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:11353
unstable_runWithPriority @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/scheduler/cjs/scheduler.development.js:643
runWithPriority$2 @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:11305
flushSyncCallbackQueueImpl @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:11349
flushSyncCallbackQueue @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:11338
scheduleUpdateOnFiber @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:21431
enqueueSetState @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:13100
Component.setState @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react/cjs/react.development.js:325
onStateChange @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-redux/lib/components/connectAdvanced.js:222
notify @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-redux/lib/utils/Subscription.js:30
notifyNestedSubs @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-redux/lib/utils/Subscription.js:69
onStateChange @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-redux/lib/components/connectAdvanced.js:219
notify @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-redux/lib/utils/Subscription.js:30
notifyNestedSubs @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-redux/lib/utils/Subscription.js:69
onStateChange @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-redux/lib/components/connectAdvanced.js:219
notify @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-redux/lib/utils/Subscription.js:30
notifyNestedSubs @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-redux/lib/utils/Subscription.js:69
onStateChange @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-redux/lib/components/connectAdvanced.js:219
dispatch @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/redux/lib/createStore.js:186
generalMiddleware @ /Applications/Joplin.app/Contents/Resources/app.asar/lib/BaseApplication.js:444
generalMiddleware @ /Applications/Joplin.app/Contents/Resources/app.asar/app.js:358
(anonymous) @ /Applications/Joplin.app/Contents/Resources/app.asar/lib/BaseApplication.js:381
dispatch @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/redux/lib/applyMiddleware.js:45
reduxSharedMiddleware @ /Applications/Joplin.app/Contents/Resources/app.asar/lib/components/shared/reduxSharedMiddleware.js:74
async function (async)
reduxSharedMiddleware @ /Applications/Joplin.app/Contents/Resources/app.asar/lib/components/shared/reduxSharedMiddleware.js:71
generalMiddleware @ /Applications/Joplin.app/Contents/Resources/app.asar/lib/BaseApplication.js:452
generalMiddleware @ /Applications/Joplin.app/Contents/Resources/app.asar/app.js:358
(anonymous) @ /Applications/Joplin.app/Contents/Resources/app.asar/lib/BaseApplication.js:381
refreshNotes @ /Applications/Joplin.app/Contents/Resources/app.asar/lib/BaseApplication.js:342
async function (async)
refreshNotes @ /Applications/Joplin.app/Contents/Resources/app.asar/lib/BaseApplication.js:292
generalMiddleware @ /Applications/Joplin.app/Contents/Resources/app.asar/lib/BaseApplication.js:525
async function (async)
generalMiddleware @ /Applications/Joplin.app/Contents/Resources/app.asar/lib/BaseApplication.js:452
generalMiddleware @ /Applications/Joplin.app/Contents/Resources/app.asar/app.js:358
(anonymous) @ /Applications/Joplin.app/Contents/Resources/app.asar/lib/BaseApplication.js:381
tagItem_click @ /Applications/Joplin.app/Contents/Resources/app.asar/gui/SideBar/SideBar.js:251
onClick @ /Applications/Joplin.app/Contents/Resources/app.asar/gui/SideBar/SideBar.js:301
callCallback @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:347
invokeGuardedCallbackDev @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:397
invokeGuardedCallback @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:454
invokeGuardedCallbackAndCatchFirstError @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:468
executeDispatch @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:600
executeDispatchesInOrder @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:622
executeDispatchesAndRelease @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:725
executeDispatchesAndReleaseTopLevel @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:733
forEachAccumulated @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:707
runEventsInBatch @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:750
runExtractedPluginEventsInBatch @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:881
handleTopLevel @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:5901
batchedEventUpdates @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:2344
dispatchEventForPluginEventSystem @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:5996
dispatchEvent @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:6025
unstable_runWithPriority @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/scheduler/cjs/scheduler.development.js:643
runWithPriority$2 @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:11305
discreteUpdates$1 @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:21672
discreteUpdates @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:2359
dispatchDiscreteEvent @ /Applications/Joplin.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.development.js:5979
Show 3 more frames
```

</details>

**Comment by laurent22** [comment]
Is there any error in the console? https://joplinapp.org/debugging

**Comment by viktomas** [comment]
@laurent22 Thanks for getting back to me. I linked them in the "Logfile" section, but I don't think they were relevant (no new logs were outputted during my clicking)

**Comment by laurent22** [comment]
I couldn't quite replicate this bug but I think the issue is just that the click area of the tag (and notebook) item was too narrow. So sometimes it looked like you clicked on it but you didn't. I've changed this so that the click area really take the whole space and that should fix it.

**Comment by viktomas** [comment]
@laurent22 the clicking doesn't work no matter where on the tag I click. I'm doubt that enlarging the area will fix the issue. It's a shame that I've got a lot of private notes in my DB, otherwise, I'd export it.

I played around a bit and when I added the third tag, the issue disappeared, and when I removed the third tag (leaving only two) the issue was present again.

**Comment by vgarvardt** [comment]
Reproducible in newer version

OS: Catalina 10.15.6

```
Joplin for Desktop

Copyright © 2016-2020 Laurent Cozic
Joplin 1.2.6 (prod, darwin)

Client ID: 88fe0e8275424273ada11b49ff799305
Sync Version: 2
Profile Version: 34
Keychain Supported: Yes

Revision: f5f117c (master)
```

**Comment by JoffreyLGT** [comment]
I have the same issue with only one tag in my app that become unresponsive.
The issue disappears when I restart the app, which make the error collection a bit harder.

**Comment by laurent22** [comment]
> I have the same issue with only one tag in my app that become unresponsive.

On macOS too? Catalina?

**Comment by Jun-Chenge** [comment]
I have a similar issue but in the opposite way: I have two items in my TAGS, but only the last one is clickable when I am in one of my NOTEBOOKS. After I click the last tag item, then I am able to click the first tag. I found this issue on both Ubuntu 18.04.5 and macOS High Sierra.

**Comment by JoffreyLGT** [comment]
> I have the same issue with only one tag in my app that become unresponsive.
> On macOS too? Catalina?

No, on Windows 10 running the lastest version. Got the issue on 2 computers.

**Comment by laurent22** [comment]
If one of you is familiar with HTML, could you try to open the dev tools in the Help menu when it happens, then try to find what element might be overlapping the tag element?

**Comment by vgarvardt** [comment]
@laurent22 looking at HTML I do not see anything wrong with overlapping. Which of these element is actually clickable? The only thing that looks suspicious to me so far is three onclick handlers for the tag element (the last image) and the second of them cancels event propagation, so if the third one is actually responsible for click logic then it is not reachable.

<img width="752" alt="Screenshot 2020-10-20 at 14 16 21" src="https://user-images.githubusercontent.com/173697/96585094-65cef100-12df-11eb-948a-846b1ed9960c.png">
<img width="764" alt="Screenshot 2020-10-20 at 14 17 06" src="https://user-images.githubusercontent.com/173697/96585098-6798b480-12df-11eb-8e25-a279b28966c8.png">
<img width="843" alt="Screenshot 2020-10-20 at 14 18 40" src="https://user-images.githubusercontent.com/173697/96585106-68c9e180-12df-11eb-964b-96347ce87bf3.png">

**Comment by yuichi8** [comment]
I have one tag and this happend to me.
Once the tag get to unclickable, Goto Anything (type `#tagName`) doesn't work.

```
Joplin 1.2.6 (prod, darwin) f5f117c (master)
```

**Comment by viktomas** [comment]
I can confirm that when the tag can't be clicked, it can't be accessed through "Goto Anything" command either.

I just set up local joplin development environment. On the latest commit, the issue is present (https://github.com/laurent22/joplin/commit/98f822d89cb6f5ff260080f2387010b7298f076d)

It could be reproduced by creating two notebooks, each having one note, each note having one tag. I exported the jex file:


[tag issue.jex.zip](https://github.com/laurent22/joplin/files/5421092/tag.issue.jex.zip)
 
I didn't see any console issue even when running the "Goto Anything" command.

**Comment by JunCEEE** [comment]
> I have a similar issue but in the opposite way: I have two items in my TAGS, but only the last one is clickable when I am in one of my NOTEBOOKS. After I click the last tag item, then I am able to click the first tag. I found this issue on both Ubuntu 18.04.5 and macOS High Sierra.

Update: When I added up one more tag (3 tags in total), every tag became clickable.

**Comment by viktomas** [comment]
> Update: When I added up one more tag (3 tags in total), every tag became clickable.

@JunCEEE that's not the case for me. I added a `zzz` tag and I still observe the issue happening. It happens less often (read every other time instead of every time).

**Comment by JunCEEE** [comment]
> > Update: When I added up one more tag (3 tags in total), every tag became clickable.
> 
> @JunCEEE that's not the case for me. I added a third `zzz` tag and I still observe the issue happening. It happens less often (read every other time instead of every time).

Yes, you are right. I actually have the same issue as you, but I didn't feel it in my last update. It's because of that the order of the unclickable tag changes. I found the rule goes like this: 
- Say I have 3 tags A, B and C.
- First to click C successfully, then click any notebook you have. After that, C will become unclickable, while the others are normal.
- When C is unclickable, then click any notebook you have, then click A. And then click one notebook again. Now, A becomes unclickable, and C becomes normal.

**Comment by gupta-anmol** [comment]
Facing the same issue. If I have only one tag, I am table to select it ONCE. After that if I open a notebook, clicking on that one tag doesn't do anything. If I have two tags, I have to click on the second one to make the first one responsive again.

## Environment
Joplin version: 1.2.6
Platform: macOS Catalina
OS specifics: Version 10.15.7

**Comment by bladewolf55** [comment]
I believe the issue doesn't have to do with the number of tags you have, but what the last selected tag was. I've been experiencing this for the last several versions.

## Environment
Version: Joplin Desktop v1.4.7  
Platform: Windows 10 Pro v2004

## Steps to Reproduce
1.  Open Joplin with two or more tags defined
2.  Select a tag, e.g. "home"
3.  Select a notebook, e.g. "inbox"
4.  Try to select the previously selected tag, e.g. "home"

Expected: The "home" tag opens  
Actual: No response

## More Info
If you have more than one tag defined, selecting another tag then allows selecting the intended tag.

Per the  conversation above, I opened Developer Tools and don't see anything obscuring the anchor tag. I suspect the problem is somewhere in the javascript, maybe once a tag folder is selected its onclick is disabled?

**Comment by laurent22** [comment]
Still can't replicate this. Can it have something to do with the window dimensions? Does it happen when the window is maximized for example, or a particular size?

**Comment by gupta-anmol** [comment]
> Does it happen when the window is maximized for example, or a particular size?

I tried different dimensions and also maximising the window, still facing the issue.

**Comment by JunCEEE** [comment]
I just tested with a maximized window and a normal-sized window. The problem stayed the same. The description from @bladewolf55 is the one closest to the essence. I could reproduce the problem with his steps.

**Comment by laurent22** [comment]
Yes that was it, I manage to replicate it with @bladewolf55 steps. Please give it a try in the next version and let me know if it's now working for you.

**Comment by bladewolf55** [comment]
Glad I could help.

**Comment by bladewolf55** [comment]
This is fixed in 1.4.9. Thank you!

**Comment by viktomas** [comment]
It's fixed for me in `1.4.9` as well 🎉  Thank you!

**Comment by JunCEEE** [comment]
Yes, the new 1.4.9 works. Thanks!
